lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4400,9 +4400,8 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
     // Disable the buffer's top-level cleanup (which is supposed to operate on
     // the buffer), create a cleanup for the value that carrys all child
     // cleanups.
-    auto valueCleanup = makeCleanup(adjVal, emitCleanup,
-        adjBuf.getCleanup()
-            ? adjBuf.getCleanup()->getChildren() : ArrayRef<Cleanup *>());
+    auto valueCleanup = makeCleanup(adjVal, emitCleanup, adjBuf.getCleanup()
```

Comment:
I believe this can be simplified:
```suggestion
    auto valueCleanup = makeCleanup(adjVal, emitCleanup, adjBuf.getCleanup());
```